english/dream/fly/space/space.md

```diff
@@ -6,6 +6,17 @@ So you head off to galaxy far, far away....
 
 You eventually reach a solar system with a planet that has intelligent life.
 
-While you are there, they explain that dreams are actually the real Universe 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.